include/caffe/common_layers.hpp

```diff
@@ -301,9 +301,9 @@ class MVNLayer : public Layer<Dtype> {
      const vector<bool>& propagate_down, const vector<Blob<Dtype>*>& bottom);
 
   Blob<Dtype> mean_, variance_, temp_;
-
   /// sum_multiplier is used to carry out sum using BLAS
   Blob<Dtype> sum_multiplier_;
+  Dtype eps;
```

Comment:
should be `eps_` to follow Google style guide convention for private member vars